strings/src/main/res/values/strings.xml

```diff
@@ -990,4 +990,6 @@
     <string name="not_a_directory">%s exists, but is not a directory</string>
 
     <string name="select_odk_shortcut">Select ODK Shortcut</string>
+    <string name="record_audio">Record audio</string>
```

Comment:
I spent too much time organizing this file for new strings to just pile up at the end!! 😄

Could you please move it to after the foreground recording strings? ~295. Would be great to get a comment too. `Text for a form entry setting to toggle background audio recording requested by the form.`

Reply:
Ah sorry I keep meaning to come back to these but forgetting. Definitely continue telling me off for this!